frontend/coprs_frontend/commands/add_warning_banner.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from typing import Optional
+
+import click
+from flask import render_template
+
+
+@click.command()
+@click.option("--outage_time", "-o", default=None, help="start time of the outage")
+@click.option("--ticket", "-t", default=None, help="fedora infra ticket ID")
+@click.option("--rest", default=None, help="additional code")
+def add_warning_banner(
+    outage_time: Optional[str], ticket: Optional[int], rest: Optional[str]
+) -> None:
```

Comment:
We will need to discuss the amount of explicit types in our codebase. Although I very much like static typing and the advantages it brings ... in cases like this, it is just unnecessary clutter. I will make my case in https://github.com/fedora-copr/debate and we can debate there.

PS: I am not at all saying that I don't want to use explicit types. I quite like them, even though I think python chose an ugly syntax :D

Reply:
My type hinting proposal: https://github.com/fedora-copr/debate/pull/3